src/Test/QuickCheck/All.hs

```diff
@@ -29,7 +29,7 @@ import Language.Haskell.TH
 import Test.QuickCheck.Property hiding (Result)
 import Test.QuickCheck.Test
 import Data.Char
-import Data.List
+import Data.List as List
```

Comment:
This is not correct  way to fix this compability, rather you should use

```
import Data.List (nubBy, isPrefixOf)
```

The same applies to other modules. Use explicit import list.

Reply:
Thanks!